websocket/data/operations.go

```diff
@@ -18,4 +18,5 @@ type WebSocketConfig struct {
 	AcknowledgeTimeoutInSec    int    // The duration in seconds to wait for an acknowledgement message
 	BlockingAckOnError         bool   // Set to `true` to send the acknowledgment message only if the processing part of a message succeeds. If an error occurs during processing, the acknowledgment will not be sent.
 	DropMessagesIfNoConnection bool   // Set to `true` to drop messages if there is no active WebSocket connection to send to.
+	Version                    uint32 // Defines the payload version.
```

Comment:
TBD here if we actually want it to be just an incremental value or some sort of versioning like tags on releases with minor/major changes

Reply:
added uint since we want to have version change only for major/breaking changes